fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/identifiers/NodeResourceConverter.java

```diff
@@ -33,7 +34,7 @@
  * @author cabeer
  * @since 10/15/14
  */
-public class NodeResourceConverter extends Converter<Node, FedoraResource> {
+public class NodeResourceConverter extends IdentifierConverter<Node, FedoraResource> {
```

Comment:
This isn't actually an converter of identifiers, is it? It actually retrieves `Node`s. That matters to me, because part of the impetus for this PR was exactly the difference in cost between manipulating identifiers and retrieving nodes.